webpack.common.js

```diff
@@ -43,19 +37,24 @@ module.exports = {
       ],
       dependOn: 'adhocracy4'
     },
-    unload_warning: {
-      import: [
-        './meinberlin/assets/js/unload_warning.js'
-      ],
-      dependOn: 'adhocracy4'
-    },
     dsgvo_video_embed: {
       import: [
         'dsgvo-video-embed/dist/dsgvo-video-embed.min.css',
         'dsgvo-video-embed/dist/dsgvo-video-embed.min.js'
       ],
       dependOn: 'adhocracy4'
     },
+    // these do not rely on adhocracy and adding the depend causes console error
+    // error possibly due to needing to be loaded at specific time
+    blueprint_picker: {
+      import: './meinberlin/assets/js/blueprint-picker.js'
+    },
+    documents: {
+      import: './meinberlin/apps/documents/assets/react_documents_init.js'
+    },
+    unload_warning: {
+      import: './meinberlin/assets/js/unload_warning.js'
+    },
```

Comment:
this gets ride of the annoying console errors finally, I think it's because these need to be initialised right away and do not have dependencies in adhocracy4.js but I don't know for sure